contrib/jenkins/init_build.sh

```diff
@@ -124,7 +98,6 @@ function update-dep() {
 function main() {
   update-golang || error_exit 'Failed to update golang'
   update-helm   || error_exit 'Failed to update helm'
-  update-glide  || error_exit 'Failed to update glide'
```

Comment:
@kibbles-n-bytes Please check the changes in this file.

Reply:
@MHBauer This file is no longer run for Jenkins builds, so these changes are no-ops.